cmd/git-sync/main.go

```diff
@@ -58,6 +59,11 @@ var flMaxSyncFailures = flag.Int("max-sync-failures", envInt("GIT_SYNC_MAX_SYNC_
 	"the number of consecutive failures allowed before aborting (the first pull must succeed)")
 var flChmod = flag.Int("change-permissions", envInt("GIT_SYNC_PERMISSIONS", 0),
 	"the file permissions to apply to the checked-out files")
+	
+var flWebhooks = flag.String("webhook", envString("GIT_SYNC_WEBHOOK", ""),
+	"the JSON formatted array of webhooks to be sent when git is synced")
+var flWebhookTimeout = flag.Int("webhook-timeout", envInt("GIT_SYNC_WEBHOOK_TIMEOUT", 60),
```

Comment:
Why make this a separate flag vs part of the webhook config?  or why not make the webhook config be 4 flags (--webhook-url, --webhook-method, --webhook-timeout, --webhook-success) ?

Reply:
I assume it was done because of the comment here: https://github.com/kubernetes/git-sync/issues/81#issuecomment-366018849